src/main/io/dashboard.c

```diff
@@ -71,11 +72,20 @@
 #include "sensors/sensors.h"
 
 
+PG_REGISTER_WITH_RESET_TEMPLATE(dashboardConfig_t, dashboardConfig, PG_DASHBOARD_CONFIG, 0);
+
+PG_RESET_TEMPLATE(dashboardConfig_t, dashboardConfig,
+    .device = DASHBOARD_I2C_INSTANCE,
+    .address = DASHBOARD_I2C_ADDRESS,
+);
+
 #define MICROSECONDS_IN_A_SECOND (1000 * 1000)
 
 #define DISPLAY_UPDATE_FREQUENCY (MICROSECONDS_IN_A_SECOND / 5)
 #define PAGE_CYCLE_FREQUENCY (MICROSECONDS_IN_A_SECOND * 5)
 
+static busDevice_t bus;
```

Comment:
Yes, I was in two minds about whether to do it this like that. But if we want to refactor at a future date it's just a simple search and replace to change this.